packages/stylable.io/docs/references/imports.md

```diff
@@ -6,39 +6,95 @@ layout: docs
 
 **Stylable** enables you to import other stylesheets and modules in a way that is similar to [JS Imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import). You can then use the stylesheet or module as it's been defined, or just one or more of its named values, in your own **Stylable** stylesheet.
 
-You use the **Stylable** syntax beginning with `-st-` for the `:import` config:
+## Import syntax
```

Comment:
Generally I think we should treat the old verbose syntax alive as `legacy` and refer to it as such instead of `old`